adapters/repos/db/vector/hnsw/condensor.go

```diff
@@ -65,7 +65,7 @@ func (c *MemoryCondensor) Do(fileName string) error {
 	}
 
 	if res.EntrypointChanged {
-		if err := c.SetEntryPointWithMaxLayer(int(res.Entrypoint),
+		if err := c.SetEntryPointWithMaxLayer(int64(res.Entrypoint),
```

Comment:
res.Entrypoint should be/become a `uint64`, `.SetEntryPointWithMaxLayer` should accept a `uint64`